chips/sam4l/src/adc.rs

```diff
@@ -408,12 +447,19 @@ impl hil::adc::Adc for Adc {
             self.active.set(true);
             self.continuous.set(true);
 
+            let trgsel;
+            if self.cpu_clock.get() {
+                trgsel = 1; // internal timer trigger
```

Comment:
I was having issues using ITIMER when configured to sample off of RC32K and RCSYS. Changing the value anywhere between 0x0000 and 0xffff did not seem to alter sampling rate. I will give it one more try but this was why I used continuous mode with the RC clocks.